src/Searchlight/Exceptions/DuplicateInclude.cs

```diff
@@ -8,5 +8,8 @@ public class DuplicateInclude : SearchlightException
     {
         public string Table { get; internal set; }
         public string ConflictingIncludeName { get; internal set; }
+        public string ErrorMessage { get; internal set; } = 
+            "The Searchlight model had two different included data sets with the same name. " +
+            "Searchlight requires that all included data sets have unique names.";
```

Comment:
public string ErrorMessage { get {
return $"The Searchlight model {Table} had two different included data sets with the name '{ConflictingIncludeName}'.";
}
}